CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
-# Unreleased
+# Changelog
+All notable changes to this project will be documented in this file.
 
-# v0.2.0
-* Provider is now one file, for easier distribution
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# v0.1.1
-* Removed a debug statement
+## [Unreleased]
 
-# v0.1.0
-* initial release
\ No newline at end of file
+## [0.2.0] - 2015-06-29
```

Comment:
Tag 0.2.1 is missing I think: https://github.com/cyberark/summon-keyring/releases